lib/src/command/outdated.dart

```diff
@@ -421,17 +430,16 @@ Future<void> _outputJson(
   );
 }
 
-Future<void> _outputHuman(
-  List<_PackageDetails> rows,
-  Mode mode, {
-  @required bool showAll,
-  @required bool useColors,
-  @required bool includeDevDependencies,
-  @required bool lockFileExists,
-  @required bool hasDirectDependencies,
-  @required bool hasDevDependencies,
-  @required bool showTransitiveDependencies,
-}) async {
+Future<void> _outputHuman(List<_PackageDetails> rows, Mode mode,
+    {@required bool showAll,
+    @required bool useColors,
+    @required bool includeDevDependencies,
+    @required bool lockFileExists,
+    @required bool hasDirectDependencies,
+    @required bool hasDevDependencies,
+    @required bool showTransitiveDependencies,
+    @required bool hasUpgradableResolution,
+    @required bool hasResolvableResolution}) async {
```

Comment:
Can I have trailing comma to keep alignment :D

Reply:
:) yes